### PR TITLE
documentation: #182 - Added documentation to Invoice package

### LINKED ISSUE #182: Document the files in mifopay/invoice package
**Summary:**

Document the files in mifopay/invoice package: https://github.com/openMF/mobile-wallet/tree/master/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice

### PR DESCRIPTION
**Pull request by theamritanair**
Fixes #182
- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@theamritanair Please fix the build.

**Comment by theamritanair** [comment]
@ankurs287 @luckyman20 please review this pull request.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice/ui/adapter/InvoicesAdapter.java [hunk 1]
```diff
@@ -61,19 +78,35 @@ public int getItemCount() {
...
+    /**
+     * This function returns the Invoice List.
+     * @return Invoice list of the type Invoice.
+     */
     public List<Invoice> getInvoiceList() {
         return mInvoiceList;
     }
 
+    /**
+     * This is the viewholder class which defines all the components
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice/ui/adapter/InvoicesAdapter.java]:
```suggestion
     * This is the ViewHolder class which defines all the components
```

**Comment by theamritanair** [comment]
@luckyman20 Applied changes.

**Comment by prajurock** [comment]
@theamritanair How you logging in the app??

**Comment by luckyman20** [comment]
@prajurock I have sent new credentials on gitter channel. Check it out. Sorry for the inconvenience caused.

**Comment by prajurock** [comment]
Ok

**Comment by prajurock** [comment]
@luckyman20 send here please.Not available there

**Comment by RajaVamsi11** [comment]
@prajurock What do you mean by not available? Your message is in [this](https://gitter.im/openMF/self-service-app) channel. If you want to join mobile wallet channel, please click [here](https://gitter.im/openMF/mobile-wallet)

**Comment by prajurock** [comment]
@RajaVamsi11 i am asking  credentials of mobile wallet with chirag .He told that he is posting in channel But he didn't post any.

**Comment by luckyman20** [comment]
Sorry, my bad. Too confused between both the repos. I have sent the credentials for `mifos-mobile` app on gitter channel and will try for the mobile wallet too but cannot assure anything.

**Comment by theamritanair** [comment]
@luckyman20 made the necessary changes. Please review them.

**Comment by theamritanair** [comment]
@ankurs287 can you please review this? :)

**Comment by rachittt** [comment]
Closing this because of inactivity.

**Pull request closed without merging**